PokeCharts/Client/PokeAPIClient.cs

```diff
@@ -0,0 +1,71 @@
+using System.Net.Http;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Client
+{
+    public class PokeAPIClient
```

Comment:
I think everything is nice with the client. I just have suggestions about code readability, let me know if you want to know them